### DISCUSSION TITLE
Stuck at nodeinfo

### DISCUSSION THREAD

**Discussion by deliverer33** [zwave-js/zwave-js-ui#4034] [Q&A] [answered]
I’m on the latest GA build, 14.3.5, and now having issues adding new devices from multiple manufacturers.  The devices communicate properly, ping works, no dropped packets, but nodeinfo never completes and just runs continuously.  I’m not sure if this is a bug, something with my controller ZooZ ZST39, but I cannot add new devices.  My existing devices function properly.  I recently moved into a new house and I’m rebuilding my zwave network, but I’m now running into this issue.  I also recently upgraded my controller firmware to the latest GA so that might be an issue as well.  Any thoughts on how to solve this challenge?  I’m at a loss and would really like to get my other 40+ devices added.  FYI I’ve been adding devices closest to the controller and expanding out from there.

**Comment by kpine** [comment, marked as answer]
Upgrade to the latest Z-Wave JS UI (hint: you aren't on the latest).

**Reply by deliverer33** [reply]
Weird, docker wasn’t saying there was an update but yes I forced my docker update and indeed that resolved it!  Upgraded to 14.3.6.
